test/vm.install

```diff
@@ -47,18 +47,16 @@ else
     # build an OCI image from our local repository; ouput to a local registry,
     # as rpm-ostree does not support a local podman image, and using a registry is the realistic use case
     # use the same registry volume as bots/images/scripts/bootc.setup
-    if [ "${branch%fedora*}" != "$branch" ]; then
```

Comment:
That may just have been a leftover from the time when we still had a `rhel-atomic` image?

Reply:
or rhel4edge?